services/QuillLMS/client/app/bundles/Staff/components/evidence/hints.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from "react";
+import { useQuery, useQueryClient, } from 'react-query';
+import { useHistory, } from 'react-router-dom';
+
+import Navigation from './navigation';
+
+import { HintInterface } from '../../interfaces/evidenceInterfaces';
+import { createHint, deleteHint, fetchHints, updateHint, } from '../../utils/evidence/hintAPIs';
+import { DropdownInput, Error, Spinner, Snackbar, defaultSnackbarTimeout, } from '../../../Shared/index';
+import { renderErrorsContainer } from "../../helpers/evidence/renderHelpers";
+import RuleHint from "./configureRules/ruleHint";
+import RuleHintDropdown from "./configureRules/ruleHintDropdown";
+
+const Hints = ({ location, match }) => {
+  const { params } = match;
+  const { hintId } = params;
+
+  const history = useHistory();
+
+  const newHint: HintInterface = {
```

Comment:
nbd, but this could be pulled out and exported from the Evidence constants file where we have similar constants for a new activity, new rule, etc: https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/constants/evidence.ts#L184